src/LinkAdder.php

```diff
@@ -88,7 +88,9 @@ protected function normalizeUrl(UriInterface $url): UriInterface
     protected function shouldCrawl(Node $node): bool
     {
         if ($this->crawler->mustRespectRobots()) {
-            return $this->crawler->getRobotsTxt()->allows($node->getValue());
+            if (! $this->crawler->getRobotsTxt()->allows($node->getValue())) {
```

Comment:
Let's write this as one condition, instead of a nested one.

Reply:
Done.